src/Daily-Double.js

```diff
@@ -8,14 +8,18 @@ class DailyDouble extends Clue{
 
   determineWager(turns, player, cluePoints) {
     let wagerLimit = 0;
+    this.ddPointValue = cluePoints;
+    // debugger
     if (turns > 16) {
-      this.pointValue = this.pointValue * 2;
+      this.ddPointValue = cluePoints * 2;
+      console.log(this.ddPointValue)
     }
-    if (player.score > cluePoints) {
+    if (player.score > this.ddPointValue) {
       wagerLimit = player.score
     } else {
-      wagerLimit = cluePoints;
+      wagerLimit = this.ddPointValue;
```

Comment:
Will the `ddPointValue` always be the highest remaining score on the board? If not, you may need to pass the highest remaining value on the board into this method on line 9

Reply:
Just kidding, I see how you're passing in the highest clue value in the index.js file. This parameter name `cluePoints` is a little unclear, but I see what you're doing now.

Reply:
Yes I agree, variable names are very iffy and will be changed to be more logical.